src/main/java/org/primefaces/component/cascadeselect/CascadeSelectRenderer.java

```diff
@@ -83,18 +84,18 @@ protected void encodeMarkup(FacesContext context, CascadeSelect cascadeSelect) t
 
         renderARIACombobox(context, cascadeSelect);
 
-        encodeInput(context, cascadeSelect);
-        encodeLabel(context, cascadeSelect);
+        String valueToRender = ComponentUtils.getValueToRender(context, cascadeSelect);
```

Comment:
It´s only partial solved. Only solved for SelectItems which only have a label.
It does not work when we have value and label. In this case we see after update the value instead of the label.

Reply:
Fixing with another PR